clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -2521,11 +2524,36 @@ public void testCommitOffsetMetadata() {
 
         AtomicBoolean success = new AtomicBoolean(false);
 
-        Map<TopicPartition, OffsetAndMetadata> offsets = singletonMap(t1p, new OffsetAndMetadata(100L, "hello"));
+        OffsetAndMetadata offsetAndMetadata = new OffsetAndMetadata(100L, "hello");
+        Map<TopicPartition, OffsetAndMetadata> offsets = singletonMap(t1p, offsetAndMetadata);
+        Map<TopicPartition, OffsetAndMetadata> cache = coordinator.committedOffsetsCache();
+        assertTrue(cache.isEmpty());
         coordinator.commitOffsetsAsync(offsets, callback(offsets, success));
         coordinator.invokeCompletedOffsetCommitCallbacks();
         assertTrue(success.get());
         assertEquals(coordinator.inFlightAsyncCommits.get(), 0);
```

Comment:
I know this is not your change, but please also update it. Thanks.

Reply:
You are right. I should've have dealt with this as well earlier! I'll push a change to fix this!